scripts/itkdev-docker-compose

```diff
@@ -114,6 +122,50 @@ case "$cmd" in
     open $($script_path url)
     ;;
 
+  sync)
+    itkdev-docker-compose sync:db
+    itkdev-docker-compose sync:files
+    ;;
+
+  sync:db)
+    if [ -z "${REMOTE_HOST:-}" ]; then
+      (>&2 echo "${bold}Environment variable REMOTE_HOST not defined in $dot_env_file${normal}")
+      exit 1
+    fi
+    if [ -z "${REMOTE_DB_DUMP_CMD:-}" ]; then
+      (>&2 echo "${bold}Environment variable REMOTE_DB_DUMP_CMD not defined in $dot_env_file${normal} see readme for more information about this setting")
+      exit 1
+    fi
+
+    ssh deploy@${REMOTE_HOST} ${REMOTE_DB_DUMP_CMD} | eval $(itkdev-docker-compose sql:connect)
+
+    ;;
+
+  sync:files)
+    if [ -z "${REMOTE_HOST:-}" ]; then
+      (>&2 echo "${bold}Environment variable REMOTE_HOST not defined in $dot_env_file${normal}")
+      exit 1
+    fi
+    if [ -z "${REMOTE_PATH:-}" ]; then
+      (>&2 echo "${bold}Environment variable REMOTE_PATH not defined in $dot_env_file${normal}")
+      exit 1
+    fi
+    if [ -z "${LOCAL_PATH:-}" ]; then
+      (>&2 echo "${bold}Environment variable LOCAL_PATH not defined in $dot_env_file${normal}")
+      exit 1
+    fi
+
+    REMOTE_PATH=${REMOTE_PATH%/}
+    LOCAL_PATH=${LOCAL_PATH%/}
+    excludes=''
+    for i in "${REMOTE_EXCLUDE[@]}"
+    do
+      excludes+="--exclude='${i}' "
+    done
+
+    rsync -avz ${excludes} -e ${REMOTE_HOST}:${REMOTE_PATH}/ ${docker_compose_dir}/${LOCAL_PATH}
```

Comment:
Do we need to add something along the lines of `--omit-dir-times --no-perms --no-owner --no-group` to prevent warnings/errors when `rsync`'ing?

Reply:
It should not be a problem as the files are owned by you local mac user and not the issue i vagrant where permissions is strange at times.